api-requests.js

```diff
@@ -0,0 +1,75 @@
+const request = require('request-promise-native');
+const moment = require('moment');
+
+// API Credentials
+const appId = '4586bfa0';
+const appKey = '4236304488967b5683897c8980725377';            
+
+function getBusArrivalJSON(postcode) {
+    
+    return request(`https://api.postcodes.io/postcodes/${postcode}`)
+
+    .then(body => {
+        const postcodeInfoAsJSON = JSON.parse(body);
+        
+        // Extract postcode lat lon
+        const latitude = postcodeInfoAsJSON['result']['latitude'];
+        const longitude = postcodeInfoAsJSON['result']['longitude'];
+        
+        const radius = 1000; 

```

Comment:
It's good to give numbers like this a name (arbitrary numbers in code are confusing, often referred to as "magic numbers", where "magic" means "works but is not clear why"), I'd generally expect to see them pulled out to the top of the file, as you have with your API keys.

It's also common to give your "real" constants (i.e. sort of arbitrary hard-coded values, like the radius and API credentials) an allcaps camel case name to make them stand out; `APP_ID`, `APP_KEY`, etc. It's not a universal convention though, and some people will argue heavily against it, so up to you which you prefer.